djautotask/sync.py

```diff
@@ -703,7 +769,78 @@ def _assign_field_data(self, instance, object_data):
         return instance
 
 
-class TimeEntrySynchronizer(Synchronizer):
+class NoteQueryMixin:
+    def _get_query_conditions(self, query):
+        query.open_bracket('AND')
+        query.WHERE(
+            'NoteType',
+            query.NotEqual,
+            self.at_api_client.picklist
+            ['TicketNote']['NoteType']['Workflow Rule Note - Task']
+        )
+        query.close_bracket()
+        return query
+
+
+class TicketNoteSynchronizer(BatchQueryMixin, NoteQueryMixin, Synchronizer):
```

Comment:
Most of `TicketNoteSynchronizer` and `TaskNoteSynchronizer` are nearly identical. I think it would make sense to at least move `_assign_field_data` to a parent mixin.

Reply:
Good to go!